packages/ckeditor5-fullscreen/src/handlers/classiceditor.ts

```diff
@@ -35,6 +35,8 @@ export default class ClassicEditorHandler extends AbstractEditorHandler {
 
 			this.moveToFullscreen( editorUI.getEditableElement()!, 'editor' );
 			this.moveToFullscreen( editorUIView.toolbar.element!, 'toolbar' );
+			this.generatePresenceListElement();
+			this.generateDocumentOutlineElement();
```

Comment:
Let's try to keep in the specialized classes only the code that differs between them and the rest in the abstract editor, after calling the `defaultEnable()` and before executing the configured callback. It will make it easier to spot the differences (and perhaps unify the editor types one day to get rid of them).